litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/index.tsx

```diff
@@ -55,6 +55,109 @@ interface WorkflowExperiment {
   Experiment: string;
 }
 
+let installAllExp = '';
+// Initial step in experiment
+const customSteps: Steps[][] = [
+  [
+    {
+      name: 'install-chaos-experiments',
+      template: 'install-chaos-experiments',
```

Comment:
can we take this from some constant file which everyone can track and update if needed? its very deep inside the code currently

Reply:
Better not to have this as constant since its a part of the YAML.

Reply:
yes but this is a predefined thing what user can change from editor, what all predefined values we use we could have keep at one place, to make things easier for us and community (external contributors) cc: @rajdas98

Reply:
This is just a base value, as an initial value for the YAML when it's generated. We have the image name, step name and such properties of the YAML too, we can't store them all in the constant file. It's only a one-time use sort of a string. What the users edit and is being stored in the UI is the redux state called manifest.

Reply:
yes that exactly was the point, storing all the base values separately in a constant file might be beneficial, not just for workflow but for the engine yamls too, in this way if in future argo changes it schema or we change it or we decide to go with different base values for any specific thing related to the chaos workflow or the embedded engine then it would be super easy. @S-ayanide , i am good with this PR but you may start changing or may raise another PR or discuss with the team to have such easy configuration approach. cc: @ajeshbaby